WalletWasabi.Gui/Controls/WalletExplorer/PhaseState.cs

```diff
@@ -0,0 +1,17 @@
+﻿using WalletWasabi.CoinJoin.Common.Models;
+
+namespace WalletWasabi.Gui.Controls.WalletExplorer
+{
+	public struct PhaseState
+	{
+		public PhaseState(RoundPhase phase, bool error)
```

Comment:
How about naming this like: RoundPhaseState?

Reply:
done